frameworks/cassandra/tests/conftest.py

```diff
@@ -0,0 +1,37 @@
+import os
+import json
+import pytest
+import shakedown
+
+
+@pytest.fixture(scope='session', autouse=True)
+def configure_universe(request):
+    stub_urls = {}
+
+    # prepare needed universe repositories
+    stub_universe_urls = os.environ.get('STUB_UNIVERSE_URL')
+    if not stub_universe_urls:
+        return
+    for url in stub_universe_urls.split(' '):
+        package_name = url.split('/')[4] # TODO: does this need to be a random string?
```

Comment:
But the current implementation installs with a random string, I thought.
https://github.com/mesosphere/dcos-commons/blob/master/tools/run_tests.py#L93
https://github.com/mesosphere/dcos-commons/blob/master/tools/run_tests.py#L281
https://github.com/mesosphere/dcos-commons/blob/master/tools/run_tests.py#L217

I'll have to look at your link more closely to see how these two different package names are related.